### PR TITLE
After 450px down to 360px which is the smallest mobile screen size, S…

### LINKED ISSUE #74: 🐛 Bug Report: Responsive issue
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

after 450px screen size ui is not responsive

image

### Expected Behavior

For smaller device screen the ui of  home section banner should be change and look like this
image

### Steps To Reproduce

_No response_

### Anything else?

checkout this design to create a component [Click Here](https://www.figma.com/file/ZTnjjETRtg8HNV0zufJajF/GDSC?node-id=0%3A1&t=684eNs6xDEXUA2GK-1)

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by omernadeem1070**
fix: After 450px down to 360px which is the smallest mobile screen size, Screen size UI is now responsive.

**Notes for Reviewers**

Issue #74 
This PR fixes #74 

Describe the changes you've made:
I have made it responsive from 450px down to 360px which is the smallest mobile screen size.

Screenshots for the change:
image

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
@kamalchakma1 would you like to review

**Comment by kamalchakma1** [comment]
@Yashsharma1911 yes

**Review by Soumya-Kushwaha — APPROVED** [review]

File: src/Pages/Home.js [hunk 1]
```diff
@@ -190,7 +188,7 @@ export default function Home(props) {
...
...Glc9oPbiWoLPdES/S8idofw4TLS/ijtjl3ZEAt5OzwcwiKBUAr2K+srk5/ivZ9kcaaf/yeZ7XceUgYeGa7iy63/1q3JEAMNTOBmUk5ibQjhM6HH8h/iP//A1jByxJwNkQKkM5wIr2BihLiOLvSECvki93CpSxmJtAGGBy2idy84Au8bXMr8w8RkDlv+3FEqL4gwQUXw7nTC8C74ECYYRJEkQ5Bwko/Dn9aYOxxwgLGg+ZZ5gkQeTkQ17YhJ8XAlUE/Az2f21/rKFKwz8eHJWFJEHk0wQGN7+MLxGGbxc3GsYJ2Oa6zWwf6nZHmeBORIHjgTIdfEuIHHxg8eTOnScQKG6NKAQ8H4o3x31o4f16ZIWPh2yh9SwhcgA+Duc1QVQC21qM5uT5v4aKq+yFQrbUyk1FIg8XuIRMFwXdAm+Us103aoOaqOyFQqiQ3Kkk4iqbBKdOKdBcN3Sjy4aqXiioDGxLiPKTEJAHQrW+hoM5DU5YeBLkuURRgSSETvYWtLxtbnX/Q2u6nVEaKnOhoFKTrzMRFUiCk2kitL0qXRYswXW9tr+x7CiEco0gCmNDXsGybIFuTxS3HoizzK/r76+okr2QS7C8W5BIg0f5IiIHsiBfhnUsIUp+DDJc8rxgWV7J2u423BMFREGl4VX3ZEbUiwNUoTO7Bd1l2QJCGghRIOyGRkeCIH6zd8eoDQNBFEAhXcC9Gi1s7WYqVz7BLthX0xV81DQKNgZFiSwpQn7vCMPyNTtimbmkGQ/qMfJlw/8R7uaPhhyxUCjcRvPgtJN7GRuRuwnKfgcGL44hTxHzDho/fnTMys6/Z8KtPF4SZMHgK4p45dn0eI8Q+fx+LRlraGr3B+XQ9wU7HR7OHQ7p0qfDlFahDPQGbT4LYhaU4td5EJH1BdM0/V7QepgQCPf9ParPGj7z6DktcX3D2eEimly7MfVxgY9NCoxb9ZyWMMaaWWqHy30IjxXYhpSjluevVas5WEi6trU+pUFtr8rNxjSpJwtWkb6Z1wIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8NUeHBIAAAAACPr/2hU2AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA+AUpJYZRt3AXJgAAAABJRU5ErkJggg=="/>
```
Comment (Yashsharma1911) [inline src/Pages/Home.js:191]: @omernadeem1070 this is not ideal it was used before but this way of using images makes code inconsistent can you make svg separate and then import it in Home.js

**Review by Yashsharma1911 — CHANGES_REQUESTED** [review]

File: src/Pages/Home.js (hunk 1 above)
Comment (omernadeem1070) [reply src/Pages/Home.js:191]: @Yashsharma1911 sorry can you please explain?

Comment (Yashsharma1911) [reply src/Pages/Home.js:191]: yes sure in code as you can see currently to use image we are using `img` tag and in that tag `src` attribute contain whole file data which is not ideal it is making code unreadable and longer, `src` attribute should only contain path of image not image itself so request here is to import image in `src` like `./imageName` not `data:image/png...`

Comment (omernadeem1070) [reply src/Pages/Home.js:191]: Got it thanks for the explanation @Yashsharma1911

**Review by Yashsharma1911 — APPROVED** [review]

**Comment by Yashsharma1911** [comment]
@omernadeem1070 you have done some good work here I would say

**Comment by omernadeem1070** [comment]
> @omernadeem1070 you have done some good work here I would say

Thank you.

**Pull request merged**